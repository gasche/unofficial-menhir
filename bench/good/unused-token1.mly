%token A B
%start<int> a
%%

a: {}
