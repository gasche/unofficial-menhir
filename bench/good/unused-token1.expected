File "unused-token1.mly", line 1, characters 7-8:
Warning: the token A is unused.
File "unused-token1.mly", line 1, characters 9-10:
Warning: the token B is unused.
%start a
%token B
%token A

%type <int> a
%%

a:
| 
    {}

%%



